Manager/ClientManagerInterface.php

```diff
@@ -9,4 +9,11 @@ interface ClientManagerInterface
     public function find(string $identifier): ?Client;
 
     public function save(Client $client): void;
+
+    public function remove(Client $client): void;
```

Comment:
@Allypost @spideyfusion Adding methods to an interface is a breaking change and as such this PR can be merged only for the next major version which I suppose will be tagged soon after https://github.com/trikoder/oauth2-bundle/pull/43 gets merged.

Reply:
https://semver.org/#spec-item-7
This change is backwards compatible functionality. Nothing backwards incompatible was added/changed.

Reply:
@X-Coder264 Yeah this will get merged after the version bump.

Reply:
@Allypost My point was this should be merged before the next major version gets tagged. Anyway, this discussion isn't important for this PR, @spideyfusion will take care of that :)

Reply:
@X-Coder264 Indeed. :)